lib/stoplight/data_store/redis.rb

```diff
@@ -160,7 +160,7 @@ def query_failures(config, transaction: @redis)
       def normalize_failures(failures, error_notifier)
         failures.map do |json|
           Failure.from_json(json)
-        rescue StandardError => e
```

Comment:
I mean, I prefer explicit errors, but that works 🙂

Reply:
This is an interesting one. I can easily defend both approaches, but the beauty of the “standard” gem is that, in general, it has fewer opinions on stylistic matters. I don’t quite understand why Rubocop wants you to assign any exception variable, `e`, but the “standard” gem is fine with `e` as well as with `error`. Another notable difference is that the “standard” gem doesn’t force you to hang long lines in the middle of the screen, but if you prefer, you can do that. In that sense, the “standard” gem doesn’t have any opinion on it. This is the price I'm ready to pay to stress less about things that does not metter that much